server/eventlog_test.go

```diff
@@ -690,6 +690,44 @@ func TestParseSecureBootState(t *testing.T) {
 		if !contains3PUEFI || !containsWinProdPCA {
 			t.Error("expected to see both WinProdPCA and ThirdPartyUEFI certs")
 		}
+
+		containsGCEDefaultPK := false
+		if len(msState.GetSecureBoot().GetPk().GetHashes()) != 0 {
+			t.Error("found hashes in pk")
+		}
+		for _, cert := range msState.GetSecureBoot().GetPk().GetCerts() {
+			switch c := cert.GetRepresentation().(type) {
+			case *attestpb.Certificate_WellKnown:
+				if c.WellKnown == attestpb.WellKnownCertificate_UNKNOWN {
+					t.Error(("found WellKnownCertificate_UNKNOWN in pk"))
+				}
+				if c.WellKnown == attestpb.WellKnownCertificate_GCE_DEFAULT_PK {
+					containsGCEDefaultPK = true
+				}
+			}
```

Comment:
this could be simplified:

```
certs := msState.GetSecureBoot().GetPk().GetCerts()
if len(certs) != 1 {
  t.Error()
}
wkRep, ok := certs[0].(*attestpb.Certificate_WellKnown)
if !ok {
  t.Error()
}
if wkRep.WellKnown != WellKnownCertificate_GCE_DEFAULT_PK{
  t.Error()
}
```

Same with the KEK check.

Reply:
> this could be simplified:
> 
> ```
> certs := msState.GetSecureBoot().GetPk().GetCerts()
> if len(certs) != 1 {
>   t.Error()
> }
> wkRep, ok := certs[0].(*attestpb.Certificate_WellKnown)
> if !ok {
>   t.Error()
> }
> if wkRep.WellKnown != WellKnownCertificate_GCE_DEFAULT_PK{
>   t.Error()
> }
> ```
> 
> Same with the KEK check.

Done